app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarUIView.kt

```diff
@@ -54,6 +54,9 @@ class ToolbarUIView(
             hintColor = ContextCompat.getColor(context, R.color.search_text)
 
             setOnEditListener(object : mozilla.components.concept.toolbar.Toolbar.OnEditListener {
+                override fun onCancelEditing(): Boolean {
+                    TODO("not implemented")
```

Comment:
Will this cause any problems?

Reply:
Yes, TODO() throws an exception :)

Reply:
I would like to fast-follow this with #929 landing.